aldryn_events/boilerplates/bootstrap3/static/js/addons/cl.events.js

```diff
@@ -23,56 +23,57 @@ var Cl = window.Cl || {};
             });
         },
 
-        // INFO: handle calendar
-        calendar: function (calendar) {
+        _handler: function (e) {
```

Comment:
I think you had to rewrite this to test the functionality better @Marketionist /cc @vxsx